marker/v2/schema/__init__.py

```diff
@@ -1,18 +1,28 @@
-from marker.v2.schema.blocks import Block, LAYOUT_BLOCK_REGISTRY
-from marker.v2.schema.groups import GROUP_BLOCK_REGISTRY
-from marker.v2.schema.text import TEXT_BLOCK_REGISTRY
+from enum import Enum, auto
 
 
-class _BlockTypes:
-    def __init__(self):
-        pass
-
-    def add(self, registry: dict[str, Block]):
-        for k, v in registry.items():
-            setattr(self, k, k)
-
-
-BlockTypes = _BlockTypes()
-BlockTypes.add(GROUP_BLOCK_REGISTRY)
-BlockTypes.add(TEXT_BLOCK_REGISTRY)
-BlockTypes.add(LAYOUT_BLOCK_REGISTRY)
+class BlockTypes(Enum):
```

Comment:
We may need some validation that we're not missing anything in here (like something exists in the registry but doesn't exist here)

Reply:
Thats true! We have validation here implicitly because these `BlockTypes` are the ones being used as the `block_type` in the `Node` class, and the registries grab all the nodes and extract node types.

To add a new `Node`, one would first define a `BlockTypes` value here and use that in their `Node->block_type`